packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -0,0 +1,103 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('tiny', 'large');
+const ignoreContrast = {
```

Comment:
Could we include why we need to disable color-contrast here?

Reply:
Yeah, we'll want to log issues for all the accessibility-rules we disable to follow-up and investigate them more.